pkg/api/login_oauth.go

```diff
@@ -67,7 +68,7 @@ func genPKCECode() (string, string, error) {
 	return string(ascii), pkce, nil
 }
 
-func (hs *HTTPServer) OAuthLogin(ctx *models.ReqContext) {
+func (hs *HTTPServer) OAuthLogin(ctx *models.ReqContext) response.Response {
```

Comment:
What if we don't want to return any response because the handler writes the response itself? For now we return a `nil`. We'll see how the handlers would look like in the future.